--- received
+++ proposed
@@ -132,29 +132,22 @@
       "Resource": "arn:aws:bedrock:ap-northeast-1:796575284584:knowledge-base/BEF0NPAOGN"
     },
     {
-      "Sid": "OpenSearchServerlessCollection",
-      "Effect": "Allow",
-      "Action": [
-        "aoss:CreateCollection"
-      ],
-      "Resource": "arn:aws:aoss:ap-northeast-1:796575284584:collection/*"
-    },
-    {
       "Sid": "OwnOpenSearchServerlessCollection",
       "Effect": "Allow",
       "Action": [
         "aoss:UpdateCollection",
         "aoss:DeleteCollection",
-        "aoss:TagResource",
-        "aoss:UntagResource",
         "aoss:APIAccessAll"
       ],
       "Resource": "arn:aws:aoss:ap-northeast-1:796575284584:collection/r1c8mzga2ln7rlto73a1"
     },
     {
-      "Sid": "OpenSearchServerlessPoliciesNoResourceLevelSupport",
-      "Effect": "Allow",
+      "Sid": "DenyAossProvisioningAndPolicyChangesDuringMigration",
+      "Effect": "Deny",
       "Action": [
+        "aoss:CreateCollection",
+        "aoss:TagResource",
+        "aoss:UntagResource",
         "aoss:CreateSecurityPolicy",
         "aoss:UpdateSecurityPolicy",
         "aoss:DeleteSecurityPolicy",
